accessibility-checker-extension/src/ts/util/tabMessaging.ts

```diff
@@ -33,8 +34,11 @@ export default class TabMessaging {
             var blob = new Blob([json_string], { type: "application/json" });
 
             var url = URL.createObjectURL(blob);
-            delete myMessage.report;
+            
             myMessage.blob_url = url;
+            myMessage.totalTime = myMessage.report.totalTime;
```

Comment:
This only happens on Chrome, so totalTime won't be defined above. If you're going to do this, I think you need to set the totalTime somewhere for Firefox, or else the reference to totalTime in the other file will fail.